.github/workflows/integration-tests.yml

```diff
@@ -120,4 +120,4 @@ jobs:
       - run: minikube start --namespace=booster-my-store-kubernetes
       - run: npx lerna bootstrap
       - run: npx lerna run compile
-      - run: npx lerna run integration/k8s-deploy --stream
```

Comment:
I mixed both k8s-deploy and k8s-nuke in the same job

Reply:
🤔  Just take into account that it would be harder this way to know which test failed from the PR, as we would only have one "check" for the whole k8s tests. 

It might be ok, though, just saying

Reply:
Agree! The only problem with this is that with a separate job we don't have the required services/pods deployed on that machine, so I'd need to deploy (again) + nuke @alvaroloes

Reply:
What about performing some kind of "deploy step" before testing to prepare the environment before tests? In that case, you can have tests in different jobs

Reply:
We already talked about this and we're splitting the tests on the same job but different steps